src/main/java/no/ssb/dlp/pseudo/service/pseudo/PseudoController.java

```diff
@@ -71,38 +68,51 @@ public class PseudoController {
      * Pseudonymizes a field.
      *
      * @param request JSON string representing a {@link PseudoFieldRequest} object.
-     * @return HTTP response containing a {@link ResponsePseudoField} object.
+     * @return HTTP response containing a {@link HttpResponse<Flowable>} object.
      */
     @Operation(summary = "Pseudonymize field", description = "Pseudonymize a field.")
-    @Post("/pseudonymize/field")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Post(value = "/pseudonymize/field", consumes = MediaType.APPLICATION_JSON)
     @ExecuteOn(TaskExecutors.IO)
-    public HttpResponse<ResponsePseudoField> pseudonymizeField(@Schema(implementation = PseudoFieldRequest.class) String request) {
-        PseudoFieldRequest req = Json.toObject(PseudoFieldRequest.class, request);
+    public HttpResponse<Flowable> pseudonymizeField(@Schema(implementation = PseudoFieldRequest.class) String request) {
+        try {
+            PseudoFieldRequest req = Json.toObject(PseudoFieldRequest.class, request);
+            log.info(Strings.padEnd(String.format("*** Pseudonymize field: %s ", req.getName()), 80, '*'));
+            PseudoField pseudoField = new PseudoField(req.getName(), req.getValues(), req.getPseudoFunc(), req.getKeyset());
```

Comment:
Do the PseudoField class need to store the values? I would suggest that the values are passed to the `process`-method instead.

Reply:
It does not need to store the values. I felt it was follow/read that way, and since it is just a pointer it's not going to have a performance impact. But i can change it.

Reply:
Fixed in: [Remove values from PseudoField object](https://github.com/statisticsnorway/dapla-dlp-pseudo-service/pull/49/commits/e4eb75ccbafbb84057e06839146934ae65497c2d)